splink/datasets/__init__.py

```diff
@@ -31,22 +31,120 @@ def __post_init__(self):
             )
 
 
-_splink_demo_data_dir = (
-    "https://raw.githubusercontent.com/moj-analytical-services/splink_demos/master/data"
-)
+_splink_datasets_data_dir = """https://raw.githubusercontent.com/moj-analytical-services
+                                /splink_datasets/master/data"""
 _datasets = [
     _DataSetMetaData(
         "fake_1000",
-        f"{_splink_demo_data_dir}/fake_1000.csv",
+        f"{_splink_datasets_data_dir}/fake_1000.csv",
         # hard code metadata to avoid needing to download to build docs
-        1000,
+        "1,000",
         250,
```

Comment:
Maybe 250 -> "250" to keep things consistent

Reply:
and maybe update type hints on `_DataSetMetaData`

Reply:
Yep, updated